src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -234,6 +235,9 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
   val akkaLogger = Logging(context.system, classOf[WorkflowActor])
   implicit val logger: WorkflowLogger = WorkflowLogger("WorkflowActor", workflow, Option(akkaLogger))
 
+  val workflowLogger = context.actorOf(WorkflowEventLogging.props(), "BusinessLogging")
+  workflowLogger ! SubscribeToLogging
```

Comment:
`workflowLogger` should perhaps be renamed as there is a already a `WorkflowLogger` instance created immediately above it.